packages/ckeditor5-font/src/ui/colorui.ts

```diff
@@ -37,7 +37,7 @@ import type { FontColorConfig } from '../fontconfig.js';
  * It is used to create the `'fontBackgroundColor'` and `'fontColor'` dropdowns, each hosting
  * a {@link module:ui/colorselector/colorselectorview~ColorSelectorView}.
  */
-export class ColorUI extends Plugin {
+export class FontColorUIBase extends Plugin {
```

Comment:
The proposed `FontColorUI` name is already taken in this package. Renamed to `FontColorUIBase`.